src/types.rs

```diff
@@ -292,6 +292,8 @@ pub struct RuntimeConfig {
 	pub max_kad_record_size: u64,
 	/// The maximum number of provider records for which the local node is the provider. (default: 1024).
 	pub max_kad_provided_keys: u64,
+	/// Avail account secret key. (default: None)
+	pub avail_secret_key: Option<String>,
```

Comment:
I think you can create a wrapper for key pair which will deserialize it from string which can be reused later